hail/python/hail/experimental/db.py

```diff
@@ -78,31 +87,39 @@ class DB:
     This class facilitates the annotation of genetic datasets with variant
     annotations. It accepts either an HTTP(S) URL to an Annotation DB
     configuration or a python :obj:`dict` describing an Annotation DB
-    configuration.
+    configuration. User must specify the region ('us' or 'eu') in which the cluster is
+    running if connecting to the default Hail Annotation DB.
 
     Examples
     --------
     Create an annotation database connecting to the default Hail Annotation DB:
 
-    >>> db = hl.experimental.DB()
+    >>> db = hl.experimental.DB(region='us')
     >>> mt = db.annotate_rows_db(mt, 'gnomad_lof_metrics') # doctest: +SKIP
     """
 
     _valid_key_properties = {'gene', 'unique'}
+    _valid_regions = {'us', 'eu'}
 
     def __init__(self,
                  *,
+                 region=None,
                  url=None,
                  config=None):
+        self.region = region
         if config is not None and url is not None:
             raise ValueError(f'Only specify one of the parameters url and config, '
                              f'received: url={url} and config={config}')
         if config is None:
             if url is None:
-                config_path = pkg_resources.resource_filename(__name__, "annotation_db.json")
-                assert os.path.exists(config_path), f'{config_path} does not exist'
-                with open(config_path) as f:
-                    config = json.load(f)
+                if self.region in DB._valid_regions:
+                    config_path = pkg_resources.resource_filename(__name__, "annotation_db.json")
+                    assert os.path.exists(config_path), f'{config_path} does not exist'
+                    with open(config_path) as f:
+                        config = json.load(f)
+                else:
+                    raise ValueError(f'Specify valid region parameter, received: region={self.region}. '
+                                     f'Valid regions are {DB._valid_regions}.')
```

Comment:
We should verify the region is correct even if you're using a custom-url-loaded or explicit config, right? 

Let's move this check up above all these if-statements. Something like:

```python3
if region not in DB._valid_regions:
    raise ValueError(...)
```

Reply:
Sounds good, I just moved the check up and pushed the changes.